definitions/scenarios/upgrade_to_satellite_6_10.rb

```diff
@@ -72,6 +87,7 @@ class PostMigrations < Abstract
     def compose
       add_step(Procedures::Service::Start.new)
       add_steps(find_procedures(:post_migrations))
+      add_step(Procedures::Pulp::Remove.new(:assume_yes => true))
```

Comment:
Here's the removal.  I'm still kinda 50/50 on this.  This completely wipes their pulp2 content dir.  So imagine a situation where something went really wrong, the user likely has db backups, but may not have a full /var/lib/pulp/ backup, they would be hosed.  

Curious thoughts about just printing a statement telling the user to run the command once they verifeid full operation?   To me that seems much safer.

Reply:
@ehelms curious your thoughts as well

Reply:
@jlsherrill agree, better to let users take a call to wipe out pulp2 content dir once migration is successful.

Reply:
Yes, okay so i've updated to print instructions, here's the output:

```
SNIP
Running Checks after upgrading to Satellite 6.10
================================================================================
Check number of fact names in database:                               [OK]
--------------------------------------------------------------------------------
Check whether all services are running:                               [OK]
--------------------------------------------------------------------------------
Check whether all services are running using the ping call:           [OK]
--------------------------------------------------------------------------------
Check for paused tasks:                                               [OK]
--------------------------------------------------------------------------------
Check to verify no empty CA cert requests exist:                      [OK]
--------------------------------------------------------------------------------
Check whether system is self-registered or not:                       [OK]
--------------------------------------------------------------------------------
Check if only installed assets are present on the system: 
/ Checking for presence of non-original assets...                     [OK]      
--------------------------------------------------------------------------------
Print pulp 2 removal instructions: 
======================================================
Migration of content from Pulp 2 to Pulp3 is complete
After verifying accessibility of content from clients,
it is highly recommend to run 'foreman-maintain content remove-pulp2'
This will remove Pulp 2, MongoDB, and all pulp2 content in /var/lib/pulp/content/
======================================================                [OK]
--------------------------------------------------------------------------------


--------------------------------------------------------------------------------
Upgrade finished.

```